code/mapping/src/mapping_data_integration.py

```
@@ -110,6 +119,46 @@ def entities_from_lidar_marker(self) -> List[Entity]:
+        for marker in data.markers:
+            if marker.type != Marker.CUBE:
+                rospy.logwarn(f"Skipping non-CUBE marker with ID: {marker.id}")
+                continue
+
+            # Transformation in das Koordinatensystem des Hero cars
+            points_array = np.array([[p.x, p.y, p.z] for p in marker.points])
+            points_array[:, 0] += 2
+            for i, p in enumerate(marker.points):
+                p.x, p.y, p.z = points_array[i]
+
+            width, length = calculate_marker_width_length_2d(marker.points)
```

Ruff (0.8.2)

Undefined name `calculate_marker_width_length_2d`

(F821)